aspnetcore/fundamentals/servers/kestrel.md

```diff
@@ -8,23 +8,29 @@ ms.custom: mvc
 ms.date: 04/04/2023
 uid: fundamentals/servers/kestrel
 ---
-# Kestrel web server implementation in ASP.NET Core
+# Kestrel web server in ASP.NET Core
 
 By [Tom Dykstra](https://github.com/tdykstra), [Chris Ross](https://github.com/Tratcher), and [Stephen Halter](https://twitter.com/halter73)
 
 :::moniker range=">= aspnetcore-8.0"
 
-Kestrel is a cross-platform [web server for ASP.NET Core](xref:fundamentals/servers/index). Kestrel is the web server that's included and enabled by default in ASP.NET Core project templates.
-
-Kestrel supports the following scenarios:
-
-* HTTPS
-* [HTTP/2](xref:fundamentals/servers/kestrel/http2)
-* [HTTP/3](xref:fundamentals/servers/kestrel/http3)
-* Opaque upgrade used to enable [WebSockets](xref:fundamentals/websockets)
-* Unix sockets for high performance behind Nginx
-
-Kestrel is supported on all platforms and versions that .NET Core supports.
+Kestrel is a cross-platform [web server for ASP.NET Core](xref:fundamentals/servers/index). Kestrel is the recommended server for ASP.NET Core, and it's configured by default in ASP.NET Core project templates.
+
+Some of the features of ASP.NET Core Kestrel server include:
+
+* **Cross-platform:** Kestrel is a cross-platform web server that runs on Windows, Linux, and macOS. Developers can choose their preferred operating system for hosting ASP.NET Core apps.
+* **High performance:** Kestrel is optimized to handle a large number of concurrent connections efficiently. It's ideal for building high-performance web apps and APIs.
+* **Lightweight:** Kestrel has a small footprint. It's optimized for running in resource-constrained environments, such as containers and edge devices.
+* **Secure by default:** Kestrel supports HTTPS and is hardened against web server vulnerabilities. It's a secure choice for hosting web applications.
```

Comment:
@blowdart How do you feel about this statement?

Reply:
"Supports HTTPS" seems like a minimal requirement for any web server isn't it? Would anyone wonder whether it did?

Reply:
A web server doesn't have to support TLS/HTTPS. It's quite common to have TLS termination happen at a reverse proxy and then forward the request to the app as plaintext HTTP.

https://en.wikipedia.org/wiki/TLS_termination_proxy